Source/AwsCommonRuntimeKit/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
trivial/debatable: Would it be possible to place this in root of the project instead of Source/*?

Reply:
I guess different modules might require different privacy manifest so it might make more sense to keep it here.